Dockerfile.moc-xdmod

```diff
@@ -25,7 +25,16 @@ RUN yum makecache && \
     libreoffice \
     mariadb-server mariadb cronie logrotate \
     perl-Image-ExifTool php-mbstring php-pecl-apcu jq \
-    chromium-headless librsvg2-tools
+    chromium-headless librsvg2-toolsm \
+    gcc openssl-devel bzip2-devel libffi-devel
+
+RUN mkdir /python-src && \
```

Comment:
The commit message for this change says, "Remove the source build for a newer Python version", but it appears to be **adding** code to build Python from source. I am confused.

Reply:
I meant to remove it as it is simpler and less invasive to run this in a python container.